Source/Scene/Model.js

```diff
@@ -1453,6 +1453,103 @@ define([
+                        var attributeVarName;
+                        for (var attributeSemantic in primitiveAttributes) {
+                            if (primitiveAttributes.hasOwnProperty(attributeSemantic)) {
+                                attributeVarName = undefined;
+                                for (var attributeName in technique.parameters) {
+                                    if (technique.parameters.hasOwnProperty(attributeName)) {
+                                        if (defined(attributeVarName)) {
+                                            break;
+                                        }
+                                        if (technique.parameters[attributeName].semantic === attributeSemantic) {
+                                            for (var varName in technique.attributes) {
+                                                if (technique.attributes.hasOwnProperty(varName)) {
+                                                    if (technique.attributes[varName] === attributeName) {
+                                                        attributeVarName = varName;
+                                                        break;
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                                // Skip if the attribute is not used by the material, e.g., because the asset was exported
+                                // with an attribute that wasn't used and the asset wasn't optimized.
+                                if (defined(attributeVarName)) {
+                                    var a = accessors[primitiveAttributes[attributeSemantic]];
+                                    if (a.hasOwnProperty('extensions')) {
+                                        var extensions = a.extensions;
+                                        if (extensions.hasOwnProperty('WEB3D_quantized_attributes')) {
+                                            var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                            var decodeMatrix = quantizedAttributes.decodeMatrix;
+                                            decodeMatrix = decodeMatrix.map(function(val) {
+                                                return val.toFixed(16);
+                                            });
+
+                                            var newMain = "decoded_" + attributeSemantic;
+                                            var matrixName = "decode_" + attributeSemantic + "_matrix";
+                                            var decodedAttributeVarName = attributeVarName.replace('a_', 'dec_');
+
+                                            // replace usages of the original attribute with the decoded version, but not the declaration
```

Comment:
That was the problem I ran into. It does need to do something differently because the current method sets off jshint, but as of right now I don't have a better answer